src/neighbors/find_neighbors/script.py

```diff
@@ -47,12 +47,15 @@ def setup_logger():
     use_rep=par["obsm_input"],
     metric=par["metric"],
     random_state=par["seed"],
-    method="umap"
+    method="umap",
+    write_knn_indices=True
 )
 
 adata.uns[par["uns_output"]] = {
     'connectivities_key': par["obsp_connectivities"],
     'distances_key': par["obsp_distances"],
+    'knn_indices_key': par["obsm_knn_indices"],
+    'knn_distances_key': par["obsm_knn_distances"],
```

Comment:
I would propose to not add these to the data object but add arguments to the densmap component.
```suggestion
```